Gemfile

```diff
@@ -3,4 +3,5 @@
 source 'https://rubygems.org'
 
 gem 'fastlane', '~> 2.227'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 13.2'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 13.2', git: 'https://github.com/wordpress-mobile/release-toolkit.git',
+                                                    branch: 'iangmaia/fluent-po-conversion-actions'
```

Comment:
🎗️ Just adding a comment here to serve as a reminder to not merge this PR until we'll have a new version of the `release-toolkit` shipped and this `Gemfile` updated to point to it.